lib/html/template/rose-bush/index.html

```diff
@@ -12,6 +12,9 @@
 <body>
 <div class="container-fluid">
 <div class="hero-unit">
+{% if logo -%}
+<img {{logo}} />
```

Comment:
I think I'd have preferred the alternative approach as it's less prone to user error. I'll leave it to @kaday to see what she thinks about this though.